.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
```

Comment:
I don't think we have a style guide yet, actually.

Is there anything similar in nature to [Ruby's Style Guide](https://rubystyle.guide/) but for JavaScript / NodeJS?

Reply:
Yea, I am 💯 sure we can find something to append to the eslint configuration as that enforces the codestyle for the mostpart. I plan to open an issue adding lint to gh-actions and start a conversation about code style options.

Reply:
@garyanewsome have any opinions about eslint configuration and a solid codestyle? ☝️